src/machine/machinectl.c

```diff
@@ -2443,7 +2587,9 @@ static int help(int argc, char *argv[], void *userdata) {
                "  kill NAME...                Send signal to processes of a VM/container\n"
                "  copy-to NAME PATH [PATH]    Copy files from the host to a container\n"
                "  copy-from NAME PATH [PATH]  Copy files from a container to the host\n"
-               "  bind NAME PATH [PATH]       Bind mount a path from the host into a container\n\n"
+               "  bind NAME PATH [PATH]       Bind mount a path from the host into a container\n"
+               "  edit NAME...                Edit settings of one or more VMs/containers\n\n"
```

Comment:
doubled `\n` at the end.
I'd like to move one `\n` to the line `Image Commands`, then each verb line has one `\n` at the end.